### DISCUSSION TITLE
cambiare separatore di campi

### DISCUSSION THREAD

**Discussion by pigreco** [ondata/guidaVisiData#55] [Domande] [answered]
Ho aperto, con VisiData, un file `Calabria.txt` che ha come separatore di campi il `;` e successivamente lo ho salvato (`Ctrl+s`) come `calabria.csv`, il separatore è rimasto sempre il `;`, come cambiarlo in `,`?

## Ubuntu 20.04 on Windows -  saul.pw/VisiData v2.6.1

**Comment by aborruso** [comment, marked as answer]
Caro @pigreco ecco come fare: https://ondata.github.io/guidaVisiData/ricette/#cambiare-il-separatore-di-campo-di-un-csv

**Reply by pigreco** [reply]
Grazie, funziona!
